package.json

```diff
@@ -18,23 +18,24 @@
 	"license": "MIT",
 	"dependencies": {
 		"@financial-times/git": "^1.2.0",
-		"@octokit/plugin-retry": "^2.2.0",
-		"@octokit/plugin-throttling": "^2.4.0",
-		"@octokit/rest": "^16.25.1",
+		"@octokit/plugin-retry": "^3.0.5",
+		"@octokit/plugin-throttling": "^3.3.4",
+		"@octokit/rest": "^18.0.9",
 		"ansi-colors": "^3.2.4",
 		"enquirer": "2.3.0",
 		"error-subclass": "^2.2.0",
 		"get-stdin": "^6.0.0",
 		"got": "^9.6.0",
 		"immer": "^3.1.2",
+		"kind-of": "^6.0.3",
```

Comment:
This is a newly-introduced dependency but I cannot see an equivalent addition in this PR that uses this package - can we do without it?

Reply:
looks like we can!

I initially added as i was getting an error that i had to bump it because of a vulnerability and it couldn't be magically fixed by snyk..however, i have just removed it and its all good 🤔. Maybe the previously failing module has patched it?